### PR TITLE
Developing

### PR DESCRIPTION
**Pull request by Bronzza**

### CODE REVIEW

File: src/main/java/MateAcad/BasicShop/Entities/Producer.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import javax.persistence.Entity;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.Set;
+
+@Table(name = "producers")
+@Entity
+@Getter
+@Setter
+
```
Comment (Taniann) [inline src/main/java/MateAcad/BasicShop/Entities/Producer.java]: remove redundant blank line

File: src/main/java/MateAcad/BasicShop/Entities/Producer.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@Accessors(chain = true)
+public class Producer extends CreatebleEntity {
+
+    @Column(name = "name")
+    private String name;
+
+    @OneToMany(mappedBy = "producer")
+    private Set<Product> products;
```
Comment (Taniann) [inline src/main/java/MateAcad/BasicShop/Entities/Producer.java:30]: do you have overridden methods equals and hashcode in Product class?

**Review by Taniann — CHANGES_REQUESTED** [review]
please resolve comments

File: src/main/java/MateAcad/BasicShop/Entities/Producer.java (hunk 2 above)
Comment (Bronzza) [reply src/main/java/MateAcad/BasicShop/Entities/Producer.java:30]: after corrections added by lombok annotation

**Review by Taniann — APPROVED** [review]
good

**Pull request merged**